core/services/ocr2/plugins/ccip/tokendata/usdc/usdc.go

```diff
@@ -182,6 +182,10 @@ func (s *TokenDataReader) getUSDCMessageBody(ctx context.Context, msg cciptypes.
 	return parsedMsgBody, nil
 }
 
+// callAttestationApi calls the USDC attestation API with the given USDC message hash.
+// The attestation API is rate limited to 10 requests per second, with a temporary ban
+// for 5 minutes if 110 requests are made within a 10-second period.
+// Documentation: https://developers.circle.com/stablecoins/reference/getattestation
```

Comment:
```suggestion
```suggestion
// The attestation service rate limit is 10 requests per second. If you exceed 10 requests 
// per second, the service blocks all API requests for the next 5 minutes and returns an
// HTTP 429 response.
//
// Documentation:
//   https://developers.circle.com/stablecoins/reference/getattestation
//   https://developers.circle.com/stablecoins/docs/transfer-usdc-on-testnet-from-ethereum-to-avalanche
```